www/front_src/src/Resources/Details/index.test.tsx

```diff
@@ -994,15 +999,15 @@ describe(Details, () => {
     );
   });
 
-  it('displays the shortcut links when the More icon is clicked', async () => {
+  it('check link redirection for logs and report icons when are clicked', async () => {
```

Comment:
```suggestion
  it('navigates to logs and report pages when the corresponding icons are clicked', async () => {
```

That's the standard test case name format ("it does something when some action is performed")